src/Templates/src/maui-mobile/.template.config/template.json

```diff
@@ -24,12 +24,6 @@
         "description": "Overrides the $(ApplicationId) in the project",
         "datatype": "string",
         "replaces": "com.companyname.MauiApp1"
-      },
-      "mauiNuGetVersion": {
```

Comment:
I removed this from the regular MAUI template because:

1. I don't even see it used
2. It seems wrong anyway because the 'replaces' text below doesn't seem to make sense

If I've misunderstood this, I will happily add it back. (And also add it to the Blazor template.)